### PR TITLE
Issue #14 - Adding Phinx as migration manager

### PR DESCRIPTION
**Pull request by lucascavalcante**
- To create a new migration: `docker exec -it smallish-app vendor/bin/phinx create <MyNewMigration>`
- To run all migrations: `docker exec -it smallish-app vendor/bin/phinx migrate -e <environment>` (all environments are configured in phinx.php file on root folder)
- To rollback all migrations: `docker exec -it smallish-app vendor/bin/phinx rollback -e <environment> -t 0`

To check more commands available, you can visit the Phinx documentation: https://book.cakephp.org/phinx/0/en/intro.html

### CODE REVIEW

File: phinx.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+return
+[
+    'paths' => [
+        'migrations' => '%%PHINX_CONFIG_DIR%%/db/migrations',
+        'seeds' => '%%PHINX_CONFIG_DIR%%/db/seeds'
+    ],
+    'environments' => [
+        'default_migration_table' => 'phinxlog',
+        'default_environment' => 'development',
+        'production' => [
```
Comment (dersonsena) [inline phinx.php:15]: Hello @lucascavalcante, I think we can add the `$_ENV` variables in the `production` section as well to be used in future deployment processes. What do you think?

**Comment by dersonsena** [comment]
Hey @lucascavalcante , first of all thanks for your contribution =).

File: composer.json [hunk 2]
```diff
@@ -23,9 +23,9 @@
     },
     "require": {
         "php": "^8.0",
+        "ext-intl": "*",
```
Comment (dersonsena) [inline composer.json:26]: @lucascavalcante did you change the `ext-intl` order just to organize? I wanted understand what did you think

**Review by dersonsena — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (lucascavalcante) [reply composer.json:26]: My linter put it on alphabetical order.

File: phinx.php (hunk 1 above)
Comment (lucascavalcante) [reply phinx.php:15]: Done!

File: composer.json (hunk 2 above)
Comment (dersonsena) [reply composer.json:26]: Ok. Thanks for clarify =)

**Review by dersonsena — APPROVED** [review]

**Pull request merged**